pingdom/contact_types.go

```diff
@@ -0,0 +1,75 @@
+package pingdom
+
+import (
+	"fmt"
+	"sort"
+	"strconv"
+)
+
+// Contact represents a Pingdom contact
+type Contact struct {
+	Name               string            `json:"name"`
+	Email              string            `json:"email,omitempty"`
+	Cellphone          string            `json:"cellphone,omitempty"`
+	Countryiso         string            `json:"countryiso,omitempty"`
+	CountryCode        string            `json:"countrycode,omitempty"`
+	Defaultsmsprovider string            `json:"defaultsmsprovider,omitempty"`
+	Directtwitter      bool              `json:"directtwitter,omitempty"`
+	Twitteruser        string            `json:"twitteruser,omitempty"`
+	Iphonetokens       string            `json:"iphonetokens,omitempty"`
+	Androidtokens      string            `json:"androidtokens,omitempty"`
+	Paused             bool              `json:"paused,omitempty"`
+	RequestHeaders     map[string]string `json:"requestheaders,omitempty"`
```

Comment:
What is this requestheaders field? I do not see in the struct of the contact, and it does not seem to be related to a contact at all. 

Maybe we added this by accident?